gpjax/__init__.py

```diff
@@ -19,7 +19,28 @@
     Prior,
     construct_posterior,
 )
-from gpjax.kernels import *
+from gpjax.kernels import (
+    RBF,
+    RFF,
+    AbstractKernel,
+    BasisFunctionComputation,
+    ConstantDiagonalKernelComputation,
+    DenseKernelComputation,
+    DiagonalKernelComputation,
+    EigenKernelComputation,
+    GraphKernel,
+    Linear,
+    Matern12,
+    Matern32,
+    Matern52,
+    Periodic,
+    Polynomial,
+    PoweredExponential,
+    ProductKernel,
+    RationalQuadratic,
+    SumKernel,
+    White,
+)
```

Comment:
i'm assuming this is some kind of auto-expander? this is one of those rare cases where I'd prefer the `import *` syntax...